### PR TITLE
Upgrade embedded Kotlin to 1.8.22

### PR DESCRIPTION
**Pull request by eskatos**
See https://github.com/JetBrains/kotlin/releases/tag/v1.8.22

### CODE REVIEW

**Comment by eskatos** [comment]
@bot-gradle test RFN AFT AST APT

**Review by lkasso — APPROVED** [review]
One very small change requested. Docs look good otherwise.

File: subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc [hunk 1]
```diff
@@ -81,9 +81,10 @@ Since the artifact view was created before the attributes were added to the conf
 Some builds may have worked around this by also putting the additional attributes into the artifact view. This is no longer necessary.
 
 [[kotlin_1_8.21]]
-==== Upgrade to Kotlin 1.8.21
+==== Upgrade to Kotlin 1.8.22
 
-The embedded Kotlin has been updated to link:https://github.com/JetBrains/kotlin/releases/tag/v1.8.21[Kotlin 1.8.21].
+The embedded Kotlin has been updated to link:https://github.com/JetBrains/kotlin/releases/tag/v1.8.22[Kotlin 1.8.22].
+See also release notes for link:https://github.com/JetBrains/kotlin/releases/tag/v1.8.21[Kotlin 1.8.21].
```
[the remark this answers is not in the document]
Comment (eskatos) [reply subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc]: Thanks!

**Review by jbartok — APPROVED** [review]

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Pull request merged**